test-app/src/main/java/com/avito/android/ui/TabLayoutActivity.kt

```diff
@@ -0,0 +1,19 @@
+package com.avito.android.ui
+
+import android.os.Bundle
+import android.support.design.widget.TabLayout
+import android.support.v7.app.AppCompatActivity
+
+class TabLayoutActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_tab_layout)
+
+        findViewById<TabLayout>(R.id.tabs).apply {
+            (1..4).forEach {
+                addTab(newTab().setText("$it"))
```

Comment:
TabLayoutElement is essential, when the layout is scrollable (`tabLayout.setTabMode(TabLayout.MODE_SCROLLABLE)`) and the desired tab does not exist yet. 

I wish there was a test, checking that we can select the invisible item: for example, set 1000 tabs and select 789 element=)